.circleci/config.yml

```diff
@@ -80,8 +78,40 @@ jobs:
       - *fetch-xcode-logs
       - *store-artifacts
 
-  spm-linux-build-and-test:
+  spm-linux-4.0-build-and-test:
+    <<: *linux-config
+    docker:
+      - image: norionomura/jazzy:swift-4.0
+    steps:
+      - *prepare-storage
+      - checkout
+      - *restore-gems
+      - *install-gems
+      - *store-gems
+      - run:
+          name: Run all tests
+          command: bundle exec rake spm:test
+      - *store-artifacts
+
+  spm-linux-4.1-build-and-test:
     <<: *linux-config
+    docker:
+      - image: norionomura/jazzy:swift-4.1
+    steps:
```

Comment:
You could create a YAML alias on line 85 (`&spm-steps`) and reference it here (`*spm-steps`) instead of duplicating them

Reply:
Right.